packaging/format-docs.py

```diff
@@ -70,6 +75,13 @@ def format_docs(version, collectionName, types):
                 print(f'| -------- | ------------- | ---- | ----------- |')
 
                 for prop in currentType["Properties"]:
+
+                    # Use the user-friendly type name unless we're certain this is a known enum,
+                    # in which case get a link to the enum's definition.
+                    typeName = prop["UserFriendlyType"]
+                    if prop["InternalType"] in enumNames:
+                        typeName = format_type_name(prop["InternalType"], True)
```

Comment:
Perhaps move the word `enum` to here? "Enum: PlayerRelationships"

<img width="502" alt="Screenshot 2022-09-12 at 19 50 28" src="https://user-images.githubusercontent.com/37534529/189711379-e10e30a8-f2da-48e2-869e-5eb16420b333.png">

Reply:
I don't think that goes too great with the other requested changes. Let's see how they go first.